tests/test_flirt.py

```diff
@@ -0,0 +1,76 @@
+import vivisect.const as v_const
+from fixtures import sample_038476
+
+from viv_utils.flirt import *
+
+
+def test_invalid_function(sample_038476):
+    """
+    test an invalid function address
+    """
+    # this is an address that is not a function
```

Comment:
I think there's a lot of duplication here with the name and two comments. I feel like the name and maybe the # comment is enough.